packages/start/src/server-handler/index.tsx

```diff
@@ -1,12 +1,191 @@
 /// <reference types="vinxi/types/server" />
-import { eventHandler, toWebRequest } from 'vinxi/http'
-import { handleRequest } from './handler'
+import { eventHandler, sendStream, setHeaders, toWebRequest } from 'vinxi/http'
+import invariant from 'vinxi/lib/invariant'
+import { getManifest } from 'vinxi/manifest'
+import { isValidElement } from 'react'
+import {
+  serverFnPayloadTypeHeader,
+  serverFnReturnTypeHeader,
+} from '../constants'
+import type { H3Event } from 'vinxi/server'
+import type { Stream } from 'stream'
 
 export default eventHandler(handleServerAction) as any
 
-export async function handleServerAction(event: any) {
+async function handleServerAction(event: H3Event) {
   const request = toWebRequest(event)
-  return await handleRequest(request)
+  const method = request.method
+  const url = new URL(request.url, 'http://localhost:3000')
+  const search = Object.fromEntries(
```

Comment:
Does this localhost URL matter here?

Reply:
Not really. It's just a placeholder to make it a valid URL.